.moai/specs/SPEC-V3R3-RETIRED-AGENT-001/acceptance.md

```
@@ -0,0 +1,509 @@
+
+**Test Anchor**: `internal/cli/launcher_worktree_validation_test.go TestPathTemplateRejectsNonStringValue`
+
+---
+
+### AC-RA-07: retired-rejection guard returns proper JSON + exit 2 (REQ-RA-004, REQ-RA-007)
+
+**Given** the M3 implementation deployed
+**And** stdin JSON `{"agentType": "manager-tdd", "agentName": "manager-tdd", "agent_id": "test-uuid-123"}`
+
+**When** the developer runs:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)